DICOMPlugins/DICOMTID1500Plugin.py

```diff
@@ -127,9 +141,20 @@ def sortReportsByDateTime(self, uids):
   def getDateTime(self, uid):
     filename = slicer.dicomDatabase.fileForInstance(uid)
     dataset = dicom.read_file(filename)
-    seriesDate = dataset.SeriesDate
-    seriesTime = dataset.SeriesTime
-    return datetime.datetime.strptime(seriesDate+seriesTime, '%Y%m%d%H%M%S')
+    if hasattr(dataset, 'SeriesDate') and hasattr(dataset, "SeriesTime"):
+      date = dataset.SeriesDate
+      time = dataset.SeriesTime
+    elif hasattr(dataset, 'StudyDate') and hasattr(dataset, "StudyTime"):
+      date = dataset.StudyDate
+      time = dataset.StudyDate
+    else:
+      date = "19630524"
```

Comment:
I changed the default to be empty strings when the value is unknown, which is what the [standard dictates](https://dicom.innolitics.com/ciods/rt-dose/general-study/00080020).  This should be fine since the string is only used for sorting.